src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,213 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars(manufacturer_id, model) VALUES(?,?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement = connection
+                         .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setLong(1, car.getManufacturer().getId());
+            createStatement.setString(2, car.getModel());
+            createStatement.executeUpdate();
+            ResultSet resultSet = createStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't create car = " + car, throwables);
+        }
+        createReferenceToCarDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        Car car = null;
+        String query = "SELECT c.id as car_id, manufacturer_id, model, name, country "
+                + "FROM cars c JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.is_deleted = false AND c.id = ?";
```

Comment:
```suggestion
                + "WHERE c.is_deleted = FALSE AND c.id = ?";
```
if you use upper case, then use it everywhere, the same for other queries